packages/smart-accounts/src/viemBundlerFactory.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  createBundlerClient as viemCreateBundlerClient,
+  bundlerActions,
+} from "viem/account-abstraction";
+import { getBlock } from "viem/actions";
+import { fromHex, type Hex } from "viem";
+
+export const DEFAULT_ENTRY_POINT_ADDRESS =
+  "0x5ff137d4b0fdcd49dca30c7cf57e578a026d2789";
```

Comment:
We'll need to support both EP v0.6 and v0.7 (also eventually v0.8). I believe viem's bundler client should already have structure for getting EP versions and default addresses, so we don't need to redefine them.